src/peer_channels.rs

```diff
@@ -217,9 +217,12 @@ impl Context {
         {
             RelayMode::Custom(RelayMap::from_url(RelayUrl::from(relay_url)))
         } else {
-            // FIXME: this should be RelayMode::Disabled instead.
-            // Currently using default relays because otherwise Rust tests fail.
-            RelayMode::Default
+            #[cfg(test)]
+            {
+                RelayMode::Default
+            }
+            #[cfg(not(test))]
+            RelayMode::Disabled
```

Comment:
you should at least print a big warning in this case, given that iroh won't work in this setup with deltachat currently

Reply:
Commented about this in the issue aswell